moveit_ros/hybrid_planning/local_planner/local_planner_component/src/local_planner_component.cpp

```diff
@@ -181,13 +183,18 @@ bool LocalPlannerComponent::initialize()
         moveit::core::RobotState start_state(planning_scene_monitor_->getRobotModel());
         moveit::core::robotStateMsgToRobotState(msg->trajectory_start, start_state);
         new_trajectory.setRobotTrajectoryMsg(start_state, msg->trajectory);
-        trajectory_operator_instance_->addTrajectorySegment(new_trajectory);
+        *local_planner_feedback_ = trajectory_operator_instance_->addTrajectorySegment(new_trajectory);
```

Comment:
Not sure sharing and initializing the feedback like this is ideal, but I don't have a better idea right now. Also, feedback messages like this should probably not be empty making the check below unnecessary.

Reply:
Maybe it should not be empty (e.g. it contains some `SUCCESS` feedback or something), but we might want to check anyway so we don't publish every cycle. This is running pretty fast, and publishing feedback here triggers a reaction by the `HybridPlanningManager`. The reaction could be trivial for trivial feedback, but it is still a lot of activity (i.e. ROS messages and plugin class function calls) for a message we care about like 1% of the time

Reply:
I agree with Adam that it would create a message overhead when we would send "Null-feedback". Do you think it would be better to initialize the feedback and check for some kind of null-value within the local planner component before sending
 @henningkayser?